roles/upload-ansible-collection-fork/tasks/publish-collection-with-ansible-devel.yml

```diff
@@ -1,10 +1,4 @@
 ---
-- name: Run our-ensure-python role
-  include_role:
-    name: our-ensure-python
-  vars:
-    ensure_python__version: "3.11"
-
 - name: Run ensure-virtualenv role
   include_role:
     name: ensure-virtualenv
```

Comment:
This would still fail because https://opendev.org/zuul/zuul-jobs/src/branch/master/roles/ensure-virtualenv/tasks/RedHat.yaml is using sudo. I think the only reason it's passing in your test is because virtualenv is already installed so it just skips the install step.

Reply:
updated